os/include/debug.h

```diff
@@ -724,6 +724,12 @@ int get_errno(void);
 #define pgllvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_SYSTEM_FAULTS
```

Comment:
Let's add this in else statement, line 1033.

Reply:
Done.